### PR TITLE
Replace bigint(BigInt) to JSBI

### PR DESCRIPTION
**Pull request by MichaelKim20**
When SDK's PR https://github.com/bpfkorea/boa-sdk-ts/pull/73 is merged, all data type `bigint` will be changed to `JSBI` in the block-related data structure included in SDK.
I created a PR that changes all data type `bigint`  from Stoa's code to JSBI according to the change.

https://github.com/bpfkorea/boa-sdk-ts/issues/72

When SDK's PR https://github.com/bpfkorea/boa-sdk-ts/pull/73 is merged, I will change this PR from draft to formal PR.

### CODE REVIEW

File: src/Stoa.ts [hunk 1]
```diff
@@ -200,14 +189,14 @@ class Stoa extends WebService
...
                     let target_height: Height = new Height(row.height);
                     let result_preimage_hash = new Hash(Buffer.alloc(Hash.Width));
-                    let avail_height: bigint = BigInt(row.avail_height);
+                    let avail_height = JSBI.BigInt(row.avail_height);
 
                     // Hashing preImage
-                    if (target_height.value >= avail_height &&
-                        (avail_height + BigInt(preimage_distance)) >= target_height.value)
+                    if (JSBI.greaterThanOrEqual(target_height.value, avail_height) &&
+                        JSBI.greaterThanOrEqual(JSBI.add(avail_height, JSBI.BigInt(preimage_distance)),target_height.value))
```
Comment (TrustHenry) [inline src/Stoa.ts]:
```suggestion
                        JSBI.greaterThanOrEqual(JSBI.add(avail_height, JSBI.BigInt(preimage_distance)), target_height.value))
```

File: src/Stoa.ts [hunk 2]
```diff
@@ -293,15 +282,15 @@ class Stoa extends WebService
...
-                    let target_height: Height = new Height(BigInt(row.height));
+                    let target_height: Height = new Height(JSBI.BigInt(row.height));
                     let result_preimage_hash = new Hash(Buffer.alloc(Hash.Width));
-                    let avail_height: bigint = BigInt(row.avail_height);
+                    let avail_height = JSBI.BigInt(row.avail_height);
                     // Hashing preImage
-                    if (target_height.value >= avail_height &&
-                        avail_height + BigInt(preimage_distance) >= target_height.value)
+                    if (JSBI.greaterThanOrEqual(target_height.value, avail_height) &&
+                        JSBI.greaterThanOrEqual(JSBI.add(avail_height, JSBI.BigInt(preimage_distance)),target_height.value))
```
Comment (TrustHenry) [inline src/Stoa.ts]:
```suggestion
                        JSBI.greaterThanOrEqual(JSBI.add(avail_height, JSBI.BigInt(preimage_distance)), target_height.value))
```

**Review by TrustHenry — APPROVED** [review]
LGTM, just two styles

**Comment by TrustHenry** [comment]
There is no commit message. only title

File: src/Stoa.ts (hunk 2 above)
Comment (MichaelKim20) [reply src/Stoa.ts]: I changed it

File: src/Stoa.ts (hunk 1 above)
Comment (MichaelKim20) [reply src/Stoa.ts]: I changed it

**Comment by MichaelKim20** [comment]
> There is no commit message. only title


I changed the commit message

**Review by TrustHenry — APPROVED** [review]

**Pull request merged**